src/EprPrnIntegration.Api/UpdateProducersFunction.cs

```diff
@@ -67,6 +68,22 @@ public async Task Run([TimerTrigger("%UpdateProducersTrigger%")] TimerInfo myTim
         }
     }
 
+    private void LogCustomEvents(List<UpdatedProducersResponseModel> updatedEprProducers)
+    {
+        foreach (var producer in updatedEprProducers)
+        {
+            Dictionary<string, string> eventData = new()
+                {
+                    { "Organization name", producer.ProducerName },
+                    { "Organisation ID", producer.OrganisationId.ToString() },
```

Comment:
should this be externalId? I reckon organisaitonId gets mapped to numeric Id column in db

Reply:
Discussed with Suketu he said it should be reference number pushed the changes and added comment on ticket